lib/gui/lvgl/Kconfig

```diff
@@ -21,15 +24,17 @@ config APP_LINK_WITH_LVGL
 	  disabled if the include paths for LVGL are causing aliasing
 	  issues for 'app'.
 
+config LVGL_USE_FILESYSTEM
+	bool "Enable file system"
+	depends on FILE_SYSTEM
+	default y if FILE_SYSTEM
+	help
+	  Enable LittlevGL file system
+
 rsource "Kconfig.graphical"
 rsource "Kconfig.memory"
 rsource "Kconfig.input"
-rsource "Kconfig.feature"
-rsource "Kconfig.image"
-rsource "Kconfig.debug"
-rsource "Kconfig.themes"
-rsource "Kconfig.fonts"
-rsource "Kconfig.text"
-rsource "Kconfig.objects"
+
+source "../modules/lib/gui/lvgl/Kconfig"
```

Comment:
we can't do that: downstream projects not cloning this module will fail to build. A copy needs to be kept in Zephyr's repository (this shows that Zephyr _modules_ are not that _modular_)

Reply:
Done.